src/components/Footer.jsx

```diff
@@ -35,7 +36,8 @@ const Footer = () => (
           >
             Creative Commons CC0 1.0 Universal Public Domain Dedication
           </a>
-          . Published <a href="/vocabulary">Vocabulary</a>.
+          . Published <a href="/vocabulary">Vocabulary</a>. &nbsp; Sinopia v
+          {Package.version}
```

Comment:
since the app version code is only used in a single place, I conferred with @justinlittman and we decided to use the same approach as in `NewsItem.jsx` and have the Footer component get the version number directly from package.json.

If there is some performance problem with this approach, then we can keep this code and `NewsItem.jsx` and the footer should retrieve the app version from state.